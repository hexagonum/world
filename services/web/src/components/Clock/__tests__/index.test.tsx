import { render } from '@testing-library/react';
import { Clock } from '..';

describe('Clock', () => {
  it('to match snapshot', () => {
    const { container } = render(<Clock />);
    expect(container).toMatchSnapshot();
  });
});
