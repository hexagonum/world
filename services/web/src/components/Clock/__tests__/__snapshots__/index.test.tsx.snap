// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Clock to match snapshot 1`] = `
<div>
  <div
    class="w-full flex justify-between items-center"
  >
    <p>
      Timezone: 
      0
    </p>
    <p
      class="font-bold"
    >
      12:01:05
    </p>
  </div>
</div>
`;
